templates/overrides/search/block--facets-summary-block.html.twig

```diff
@@ -1,9 +1,9 @@
 {#
 /**
  * @file
- * Theme block implementation to display facets summary.
+ * Override for blocks with plugin id = 'facets_summary_block'.
  *
- * @see ./core/themes/stable/templates/block/block.html.twig
+ * @see \Drupal\facets_summary\Plugin\Block\FacetsSummaryBlock
```

Comment:
These comments had changed in 1.x since before last rebase.
Please decide if my version or the 1.x version is preferable.